torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -175,6 +175,100 @@ std::vector<Node*> CreateQuantizedWeights(
   return {const_node_1, const_node_2, const_node_3};
 }
 
+std::vector<Node*> CreateQuantizedWeights(
```

Comment:
Can you add comments to this and the other function that has the same name on line 132 explaining the differences between the two? Or maybe name them differently.

Reply:
forgot to remove old code ...